app/services/daily_difference_calculator.rb

```diff
@@ -13,21 +13,23 @@ def differences
 
   private
 
-  HoodSummary = Struct.new(:name, :yesterdays_valuation, :todays_valuation, :valuation_difference, :house_count, :valuation_date)
+  HoodSummary = Struct.new(:name, :yesterdays_valuation, :todays_valuation, :valuation_difference, :house_count, :valuation_date, :average_house_diff, keyword_init: true)
```

Comment:
What's `keyword_init` about?

Reply:
Let's you use keyword arguments instead!